### ISSUE TITLE
[Implement] - Google 2 Factor

### ISSUE THREAD

**Issue by conconnl** [joomla/showcase.joomla.org#1]
Please make sure "Two Factor Authentication - Google Authenticator" is working.

**Comment by sebastienlapoux** [comment]
Thanks @conconnl .

Enabled. My superuser account has 2 factor enabled.

If you edit a user you will see the message
![image](https://user-images.githubusercontent.com/13470265/79846915-4fdb8480-83bf-11ea-88d3-69adba0e300e.png)

To enable the Two Factor Authentication on a user account, please on user manager check the checkbox of a user and then click on edit icon. It will allow to edit the user with Joomla view and to access the Two Factor Authentication tab.
![image](https://user-images.githubusercontent.com/13470265/79846096-4bfb3280-83be-11ea-94f2-451216b05512.png)

Thanks to close the ticket if it's ok for you.

**Comment by conconnl** [comment]
The login form popup doesn't support 2FA.
The official J! link in the footer does: https://showcase.joomla.org/user-login.html

<img width="653" alt="image" src="https://user-images.githubusercontent.com/11913918/79847496-1a836680-83c0-11ea-9aa5-75ce2aedecf4.png">

**Comment by sebastienlapoux** [comment]
yes and related to the issue https://github.com/joomla/showcase.joomla.org/issues/8, please test on admin or on the login link at the top of https://showcase.joomla.org/register.html

**Comment by conconnl** [comment]
I can't find the ability to configure 2FA on the front-end.
Please add this when you have time, so users have this ability available.

**Comment by sebastienlapoux** [comment]
Right. Ok.

**Comment by sebastienlapoux** [comment]
@sebastienheraud to discuss

**Comment by conconnl** [comment]
How is it going?

**Comment by conconnl** [comment]
Is this finalized in the meantime?

**Comment by sebastienlapoux** [comment]
No. It requires some dev to add 2FA fields into the SEBLOD user profile on front-end. Is it really important about registered users who are not saving critical personal data? Where as "admin" people have 2FA enabled.

**Comment by conconnl** [comment]
It has nothing to do with personal data but just plain security.
We have to provide the best services to our users and as the core already supports 2FA for years, it's not something unique or special That SEBLOD doesn't has this by default and overrides the default com_users usage, just makes it more work for you. The request was done on April 20th, 2020 we are almost 9 months in the future, I would expect that would have been a long enough period to add the default com_users fields in the system.

**Comment by vijay938** [comment]
I want to work on this issue

**Comment by conconnl** [comment]
@sebastienheraud @sebastienlapoux 
Still nothing?

**Comment by sebastienlapoux** [comment]
Currently no. Sorry for the timeline but always not planed on SEBLOD side. Concerns just front/registered users.

**Comment by itssiddhantjain** [comment]
I would like to work on this issue. Please assign me this issue.
